docs/sphinx/source/whatsnew/v0.9.5.rst

```diff
@@ -29,6 +29,8 @@ Enhancements
   :py:func:`pvlib.bifacial.infinite_sheds.get_irradiance_poa`
   to enable use of the hay-davies sky diffuse irradiance model
   instead of the default isotropic model. (:pull:`1668`)
+* Update the ADR inverter database by appending the ADR equivalents of all
+  inverters in the current (2019) Sandia inverter database. (:pull:`1695`)
```

Comment:
Did you mean "CEC inverter database"?

Reply:
Actually I meant "Sandia inverter model database".  I'll fix that.

Reply:
Ah, right. The CEC file contains parameters for the Sandia model. Glad that's not at all confusing :)